sidebarNode.ts

```diff
@@ -151,5 +123,47 @@ export const sidebarNode: SidebarsConfig = {
       label: 'gRPC Proxy',
       id: 'node/grpc',
     },
+    {
+      type: 'category',
+      label: 'Reference',
+      description: 'Reference.',
+      collapsible: false,
```

Comment:
By default I'd collapse this entry.

```suggestion
      collapsible: true,
```